### PR TITLE
Edit docs

### PR DESCRIPTION
**Pull request by apoorvsadana**
Update the `README.md` and `getting-started.md` files

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -11,45 +11,64 @@ then run:
...
-debug logs. It can also be specified via the `RUST_LOG` environment variable.
-For example:
+### Interacting with the node
 
-```sh
-RUSTLOG=runtime=info cargo run --release -- --dev
-```
+Madara is compatible with the Starknet
+[spec](https://github.com/starkware-libs/starknet-specs) which means all tooling
+around Starknet (starknet-js, starknet-rs, wallets etc.) can be used out of the
```
Comment (tdelabro) [inline docs/getting-started.md:35]: `wallets, etc.`

**Review by tdelabro — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -35,61 +35,86 @@
...
+Welcome to **Madara**, the modular stack to build chains using
+[Cairo](https://book.cairo-lang.org/title-page.html) and the
+[Starknet](https://www.starknet.io/) technology. Apps like dYdX V3, Immutable
+and Sorare have been using StarkEx for scaling for a while and now with Madara,
+it's open source for everyone to use.
 
-Built on the robust Substrate framework and fast, thanks to Rust 🦀, Madara
-delivers unmatched performance and scalability to power your Starknet-based
-Validity Rollup chain.
+Madara is built on the Substrate framework which not only makes it modular but
```
[the remark this answers is not in the document]
Comment (apoorvsadana) [reply README.md]: fixed

File: docs/getting-started.md (hunk 1 above)
Comment (apoorvsadana) [reply docs/getting-started.md:35]: fixed

Comment (tdelabro) [reply docs/getting-started.md:35]: You didn't pushed i

**Review by tdelabro — APPROVED** [review]

**Pull request merged**